app/src/main/java/com/example/poste/api/poste/models/User.java

```diff
@@ -67,6 +73,18 @@ public void setPassword(String password) {
         if (!update()) { this.password = oldPassword; }
     }
 
+    /**
+     * Adds a folder to the list of folders this user can access
+     * @param folder the folder to be added to the users list of folders
+     */
+    public void addFolder (Folder folder) {
+        folders.add(folder);
+    }
+
+    public List<Folder> getFolders() {
+        return folders;
```

Comment:
Despite above comments, I don't think this requires an API call -- we can assume the data has either been already retrieved, or the user can request a data update (screen pull-down-to-refresh or similar functionality)

Reply:
Agreed